test/unit_test/instantiate_fft_tests.hpp

```diff
@@ -118,6 +118,13 @@ INSTANTIATE_TEST_SUITE_P(WorkgroupOrGlobal, FFTTest,
                              ::testing::Values(1, 128), ::testing::Values(sizes_t{8192}, sizes_t{16384}))),
                          test_params_print());
 
+// Selected individual tests in workgroup or global size range
```

Comment:
We could add them in the already existing test suite itself. 
Any particular reason you went for a new test suite ?

Reply:
The issue can be reproduced with a single test - I didn't see the need to replicate it for inplace/outofplace, different batch sizes, etc, and the test suite already takes long enough to run!

Reply:
Ah fair enough, the lesser repeated tests the Better !